lib/Cake/View/Helper/FormHelper.php

```diff
@@ -379,6 +379,13 @@ public function create($model = null, $options = array()) {
 		$this->inputDefaults($options['inputDefaults']);
 		unset($options['inputDefaults']);
 
+		if (isset($options['action'])) {
+			trigger_error('Using key `action` is deprecated, use `url` directly instead.', E_USER_DEPRECATED);
+		}
+		if (isset($options['url']) && isset($options['url']['action'])) {
```

Comment:
Hah, thx, I am not working on so old versions anymore for years, totally forgot about that. Will fix that up and see if that makes tests pass, unless you commit the fix earlier then me :)